client/my-sites/email/email-provider-features/list.js

```diff
@@ -64,7 +66,7 @@ const getTitanFeatures = ( isMonthlyProduct = true ) => {
 		translate( 'Send and receive from your custom domain' ),
 		translate( '30GB storage' ),
 		translate( 'Email, calendars, and contacts' ),
-		translate( 'One-click import of existing emails and contacts' ),
+		translate( '24/7 support via email' ),
```

Comment:
These changes are for the old `Email Comparison` page that we only show in the domain upsell flow. I'm just making sure that we use the same list of features here.